### PR TITLE
152t docs

### PR DESCRIPTION
**Pull request by mattsfuller**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.152.1-t.1.rst [hunk 1]
```diff
@@ -4,30 +4,162 @@ Release 0.152.1-t.1
...
+
+MySQL Catalogs
+^^^^^^^^^^^^^^
+MySQL catalog names are mapped to Presto schema names.
 
 
+Patches
+=======
+The following patches from 0.152.3 have been backported to 0.152.1-t.1:
+https://github.com/prestodb/presto/commit/667ccb8d88324361753155f5b53e6428474c7032
```
Comment (cawallin) [inline presto-docs/src/main/sphinx/release/release-0.152.1-t.1.rst]: also thispatch: https://github.com/prestodb/presto/commit/d3d398e467a5a887edd03c7bdca1cece24e378f6

File: presto-docs/src/main/sphinx/release/release-0.152.1-t.1.rst [hunk 2]
```diff
@@ -4,30 +4,162 @@ Release 0.152.1-t.1
...
+Decimal
+-------
+Faster Decimal implementation
+
+
+Join Filters
+------------
+Move certain filters in the WHERE clauses to be executed as part of the INNER JOIN rather than a filter after the join
+For example: ``SELECT * FROM t t1 JOIN t t2 ON t1.a = t2.a WHERE (t1.b+t2.b)*5 > 100000000``
+Code generation for joins with filters
```
Comment (cawallin) [inline presto-docs/src/main/sphinx/release/release-0.152.1-t.1.rst]: @kokosing -- should we add something here about subqueries? e.g. things that are in 152.1-t but not in 152.

File: presto-docs/src/main/sphinx/connector/blackhole.rst [hunk 3]
```diff
@@ -23,12 +23,14 @@ Configuration
 -------------
 
 To configure the Black Hole connector, create a catalog properties file
-``etc/catalog/blackhole.properties`` with the following contents:
+``/etc/opt/prestoadmin/connector/blackhole.properties`` with the following contents:
```
Comment (cawallin) [inline presto-docs/src/main/sphinx/connector/blackhole.rst:26]: very good change, thanks for remembering to change this

Comment (cawallin) [reply presto-docs/src/main/sphinx/connector/blackhole.rst:26]: there are a couple of other places to change, but I'll add a commit to your branch with them

**Review by cawallin — APPROVED** [review]
Looks good % the patch I'm going to add that switches a few other thing to the presto-admin location. I see that the presto-admin docs are already added, are they they the updated 1.5.1 docs? (cc @petroav)

File: presto-docs/src/main/sphinx/system-requirements/hardware-requirements.rst [hunk 4]
```diff
@@ -4,9 +4,14 @@ Hardware Requirements
...
+
+**Network Bandwidth**
+
+It is recommended to have 10 Gigabit Ethernet between all the nodes in the cluster.
 
 **Other Recommendations**
+
 Presto can be installed on any normally configured Hadoop cluster. YARN should
 be configured to account for resources dedicated to Presto. For example, if a
 node has 64GB of RAM, perhaps you would normally allocate 60GB to YARN.  If you
```
Comment (kbajda) [inline presto-docs/src/main/sphinx/system-requirements/hardware-requirements.rst:17]: Should we adjust the numbers here to account for our new recommendation of 64GB for Presto as stated above?

Comment (mattsfuller) [reply presto-docs/src/main/sphinx/system-requirements/hardware-requirements.rst:17]: good catch. will fix.

File: presto-docs/src/main/sphinx/release/release-0.152.1-t.1.rst (hunk 1 above)
Comment (mattsfuller) [reply presto-docs/src/main/sphinx/release/release-0.152.1-t.1.rst]: got it

**Comment by petroav** [comment]
@cawallin yea it looks like the right presto-admin docs were included. I'm judging by the bump to 1.5.1 in the example commands. I also did some grepping in presto-admin and the these docs and compared.

**Comment by mattsfuller** [comment]
merged

**Pull request closed without merging**